src/components/form/wkt-multi.tsx

```diff
@@ -54,7 +56,11 @@ export default function WKTFieldMulti(props: WKTInputProps) {
                 <WKTSearch {...props} isDisabled={isDisabled} onSave={handleOnSave} />
               </TabPanel>
               <TabPanel>
-                <WKT {...props} disabled={isDisabled} onSave={handleOnSave} />
+                {props.canDraw ? (
+                  <WKTDrawViewer {...props} disabled={isDisabled} onSave={handleOnSave} />
```

Comment:
how about something like below

```jsx
const Viewer = props.canDraw ? WKTDrawViewer : WKT

// and then use it there like below since props are same
<Viewer {...props} disabled={isDisabled} onSave={handleOnSave} />
```